h/indexer.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from h.celery import celery
+
+from h.api import storage
+from h.api.search.index import index
+from h.api.search.index import delete
+
+__all__ = (
+    'add_annotation',
+    'delete_annotation',
+)
+
+
+@celery.task
+def add_annotation(id_):
+    annotation = storage.fetch_annotation(celery.request, id_)
```

Comment:
Will work on this in a separate PR, and marking this PR as WIP until the other one is merged.